news-bundle/src/Resources/contao/templates/news/news_short.html5

```diff
@@ -16,3 +16,12 @@
   <?php endif; ?>
 
 </div>
+
+<?php
+
+// This template is used as a news list template by default
+// Ensure that there's no JSON-LD for this news article if there is a reader as that one will
+// list all the details.
+if ($this->hasReader) {
+    $this->removeSchemaOrg('NewsArticle', $this->schemaOrgId);
```

Comment:
Does this not have to be added to `news_latest` and `news_simple` as well? They are news list templates, too.

Reply:
Yes, I said it's not finished?

Reply:
https://github.com/contao/contao/pull/3119#issuecomment-868383748